ckan/templates/user/confirm_delete.html

```diff
@@ -0,0 +1,21 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _("Confirm Delete") }}{% endblock %}
+
+{% block maintag %}<div class="row" role="main">{% endblock %}
+
+{% block main_content %}
+  <section class="module col-md-6 col-md-offset-3">
+    <div class="module-content">
+      {% block form %}
+        <p>{{ _('Are you sure you want to delete user - {name}?').format(name=user_dict.name) }}</p>
+        <p class="form-actions">
+          <form id="user-confirm-delete-form" action="{{ h.url_for('user.delete', id=user_dict.id) }}" method="post">
```

Comment:
POST-form will not work withou csrf-input

Reply:
@smotornyuk sorry, forgot that CSRF is in 2.10+ now. Okay the `h.csrf_input` call has been added to the two new templates here now